vendor-extra/ContentPageBundle/composer.json

```diff
@@ -2,6 +2,7 @@
     "name": "libero/content-page-bundle",
     "description": "A Libero content page",
     "type": "symfony-bundle",
+    "version": "0.0.1",
```

Comment:
IIRC the downside of using `path` repos is it not recognising updates. Guess this gets round it? Slight annoyance having to bump on every change.

Reply:
It symlinks the whole thing, so the version only needs to be bumped once published. At that point though, it probably would be in its own repository (and not need this field)